.travis/now

```diff
@@ -51,7 +51,7 @@ _join () {
 }
 
 _build_docs () {
-    rose make-docs --strict clean html slides latexpdf
+    etc/bin/rose-make-docs --strict clean html slides latexpdf
```

Comment:
I believe that this is a largely dev-only function?

Reply:
Fine, we will probably change this in-line with https://github.com/cylc/cylc-doc/pull/26 anyway.